packages/paste-core/components/radio-group/src/Radio.tsx

```diff
@@ -86,11 +86,16 @@ const Radio = React.forwardRef<HTMLInputElement, RadioProps>(
           >
             <Box
               as="span"
-              backgroundColor={state.disabled && state.checked ? 'colorBackgroundStrongest' : 'colorBackgroundBody'}
-              borderRadius="borderRadiusCircle"
-              height="sizeSquare25"
-              width="sizeSquare25"
-            />
+              element={`${element}_CONTROL_CIRCLE`}
```

Comment:
Is this something we want to let people customize?

Reply:
We let them customize the control so we could leave it? I'm not familiar with what this line allows users to do though. @TheSisb would love some help here since we just copied this from `IconWrapper` I believe

Reply:
Would love some additional context about this change please. ☝️

Reply:
Also, could we please add a test for this? Feel free to message me and I can help if needed, we follow a pretty consistent pattern for all the customization `element` prop tests, so there are examples in the component packages.

Reply:
FWIW we allow a consumer to customize the checkmark in a checkbox. The dot in a radio button is essentially the same thing, so it makes sense to me to have a similar level customization between both checkbox and radio.

+1 on adding a test though, because as soon as we add a customizable element, we must ensure it stays.